src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Ref.sfxproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Sdk Name="Microsoft.DotNet.SharedFramework.Sdk" />
```

Comment:
Hmm would you mind explaining how this element works? Where are the imports of the props and the targets file located?

Reply:
This inserts the props and targets imports after the props and targets imports of the `Microsoft.NET.Sdk` respectively.